diff --git a/src/laserMapping.cpp b/src/laserMapping.cpp
--- a/src/laserMapping.cpp
+++ b/src/laserMapping.cpp
@@ -1115,2 +1115,2 @@ private:
-    void map_save_callback(std_srvs::srv::Trigger::Request::ConstSharedPtr req, std_srvs::srv::Trigger::Response::SharedPtr res)
+    void map_save_callback(std_srvs::srv::Trigger::Request::SharedPtr req, std_srvs::srv::Trigger::Response::SharedPtr res)
     {
